projects/element-ng/icon/icon-status.ts

```diff
@@ -0,0 +1,122 @@
+/**
+ * Copyright Siemens 2016 - 2025.
+ * SPDX-License-Identifier: MIT
+ */
+import { InjectionToken } from '@angular/core';
+import { EntityStatusType, StatusIcon } from '@siemens/element-ng/common';
+
+import {
+  elementCircleFilled,
+  elementOctagonFilled,
+  elementSquare45Filled,
+  elementSquareFilled,
+  elementStateExclamationMark,
+  elementStateInfo,
+  elementStatePause,
+  elementStateProgress,
+  elementStateQuestionMark,
+  elementStateTick,
+  elementTriangleFilled
+} from './element-icons';
+import { addIcons } from './si-icons';
+
+/**
+ * The status icon configuration.
+ *
+ * @experimental
```

Comment:
The `STATUS_ICON` is deprecated with the hint to use this injection token. However, this is marked as experimental. This does not fit. If it handled in the developer documentation? How to use the injection token.

Reply:
As mentioned below, deprecation noticed dropped.